xmppserver/src/main/webapp/server-locale.jsp

```diff
@@ -261,6 +261,17 @@
                     <label for="zh_CN">Simplified Chinese (zh_CN)</label>
                 </td>
             </tr>
+             <tr>
+                <td>
+                    <input type="radio" name="localeCode" value="fa" <%= ("fa".equals(locale.toString()) ? "checked" : "") %> id="fa" />
+                </td>
+                <td>
+                    <a href="#" onclick="document.sform.localeCode[1].checked=true; return false;"><img src="images/language_fa.gif" alt="" /></a>
```

Comment:
I do not think you need the image at all, here (this seems to be copied from the Chinese entry). The `img` tag should probably be removed. If it _is_ needed, than the file `images/language_fa.gif` should be added to this PR.